subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -151,7 +156,9 @@ static uint8_t adv_sync_pdu_init_from_prev_pdu(struct pdu_adv *pdu,
 
 	LL_ASSERT(!ext_hdr->adv_addr);
 	LL_ASSERT(!ext_hdr->tgt_addr);
+#if !defined(CONFIG_BT_CTLR_ADV_PERIODIC_ADI_SUPPORT)
 	LL_ASSERT(!ext_hdr->adi);
+#endif /* CONFIG_BT_CTLR_ADV_PERIODIC_ADI_SUPPORT */
```

Comment:
Given that `ext_hdr->adi` is always available, this could be guarded by a IS_ENABLED

Reply:
Corrected